src/lib/stack-client.js

```diff
@@ -141,6 +141,30 @@ const getApps = function () {
   )
 }
 
+/**
+ * Detail of an installed application by its slug
+ *
+ * Returns only the `data` key of the
+ * whole JSON playload from the server
+ *
+ * @function
+ * @param {string} slug 
+ * @returns {Promise} 
+ */
+const getApp = function(slug) {
+  if (!slug) {
+    throw new Error('Missing slug')
+  }
```

Comment:
Maybe have this function async ?

IMHO, instead of throwing, a Promise.reject() would be better since the normal return type is a promise and the calling code will expect a promise here. Anyway if the calling code is using await it will work.